patches/extensions-browser-extension_user_script_manager.h.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/extensions/browser/extension_user_script_manager.h b/extensions/browser/extension_user_script_manager.h
+index 4e3eaeeedfdf970277bef3d3daffcc1698c633c4..297bf07728136fddc95c2a1276d299e6ec32319f 100644
+--- a/extensions/browser/extension_user_script_manager.h
++++ b/extensions/browser/extension_user_script_manager.h
+@@ -19,6 +19,7 @@
+ namespace content {
+ class BrowserContext;
+ }
++class BraveWalletService;
```

Comment:
this can be done in a chromium_src header override

Reply:
Sure, but we still need patching for adding the friendship with BraveWalletService, so perhaps it's better to keep both bits (which are related) in one single place in this particular case?

See the commit doing this adaptation here, for extra context:
https://github.com/brave/brave-core/pull/7866/commits/4af692e0de9878eb8546cf834b3a16a9fe25c0c3

Reply:
Btw, this change is no longer happening to `extensions/browser/extension_user_script_manager.h`, but to `extensions/browser/user_script_manager.h` instead after https://github.com/brave/brave-core/pull/7866/commits/0d4882ee41b056c6817719bc642f52de79db728b